integration_tests/base_routes.py

```diff
@@ -586,6 +586,19 @@ async def post(request: Request):
         return {"status_code": 200, "body": body}
 
 
+# ==== Exception Handling ====
+
+
+@app.exception
+def handle_exception(error):
+    return {"status_code": 500, "body": f"error msg: {error}"}
+
+
+@app.get("/sync/exception")
+def sync_exception():
+    raise ValueError("value error")
```

Comment:
Hey @IdoKendo , 

Just two more suggestions - can we also test these exceptions for async code?

And can we add some test for methods with request bodies like POST, PUT ,etc?

Reply:
Sure, will add them shortly!

Reply:
@sansyrox I Added tests for async and for put and post, let me know if you think any more tests are needed.